sunpy/visualization/tests/test_drawing.py

```diff
@@ -18,6 +22,21 @@ def aia171_test_map():
     return Map(get_test_filepath('aia_171_level1.fits'))
 
 
+@pytest.fixture
+def cropped_aia193_sample_map():
+    import sunpy.data.sample
+    m = sunpy.map.Map(sunpy.data.sample.AIA_193_JUN2012)
+    return m.submap(SkyCoord(Tx=300*u.arcsec, Ty=-325*u.arcsec, frame=m.coordinate_frame),
+                    width=500*u.arcsec,
+                    height=250*u.arcsec)
+
+
+@pytest.fixture
+def euvi195_sample_map():
+    import sunpy.data.sample
+    return Map(sunpy.data.sample.STEREO_A_195_JUN2012)
+
+
```

Comment:
The figure test shouldn't use sample data (and potentially triggering downloading).  It'd be just as good to use `aia171_test_map` and a comparable map with the observer information modified.

Reply:
I'm now using a non-map WCS so this isn't an issue.